src/message.js

```diff
@@ -108,7 +112,9 @@ Globalize.prototype.messageFormatter = function( path ) {
  */
 Globalize.formatMessage =
 Globalize.prototype.formatMessage = function( path /* , variables */ ) {
-	return this.messageFormatter( path ).apply( {}, slice.call( arguments, 1 ) );
+	return ( arguments[ 1 ] && typeof arguments[ 1 ] === "boolean" ) ?
+		this.messageFormatter( path, arguments[ 1 ] ).apply( {}, slice.call( arguments, 2 ) ) :
+		this.messageFormatter( path ).apply( {}, slice.call( arguments, 1 ) );
```

Comment:
I doubt this, consider please the following case where number of formatMessage optional variables is more then one:
Globalize.loadMessages({ en: {breadcrumb: "{0} >> {1} >> {2} >> {3}" ....
Globalize( "en" ).formatMessage( "breadcrumb", "first", "second", "third" , "forth" );
Here arguments.length - 1 === 4 and your proposal will cause incorrect slicing
So probably something like this will be OK?
	return ( arguments[ 1 ]  === true ) ?
		this.messageFormatter( path, arguments[ 1 ] ).apply( {}, slice.call( arguments, 2 ) ) :
		this.messageFormatter( path ).apply( {}, slice.call( arguments, 1 ) );